data/categories.txt

```diff
@@ -2632,41 +2632,45 @@ sw:Bandali|kivuko
 fa:ترمینال کِشتی
 
 amenity-taxi|@transport
-en:3Taxi|Taxi Stand|Taxi Rank|U+1F695|U+1F696
-ru:3Такси
-bg:3такси|шофьор
-ar:سيارة أجرة|تاكسي|مواصلات|مواصلات
-cs:3Taxi
-da:3Taxi|taxa
-nl:3Taxi
-fi:3Taksi
-fr:3Taxi
-de:3Taxi|Taxistand|Taxe|ÖPNV
+en:3Taxi Stand|Taxi Rank|Taxi|U+1F695|U+1F696
```

Comment:
I would put 3 to taxi for language neutrality.

Reply:
@vng Is there a way to see Taxi Rank suggestion in the US, and Taxi Stand in the UK? If there is no one, I would suggest showing just "Taxi" hint for language neutrality. Maybe not everyone in the US knows what is Taxi Stand.